front/src/components/member/MemberUpdateModal.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <div class="d-flex flex-column modal-mask" @click.self="closeModal">
+    <VSpacer />
+    <transition name="bounce">
+      <div v-if="rendered" class="pa-6 pb-1 modal-container">
+        <VTextField
+          label="새 닉네임을 입력하세요."
+          color="grey darken-2"
+          v-model="newNickname"
+          @keypress.enter="updateMember"
+        />
+        <div class="button-box">
+          <v-btn
+            text
+            class="mb-2 text-subtitle-1 font-weight-bold grey--text text--darken-1"
+            @click.prevent="closeModal"
+          >
+            취소
+          </v-btn>
+          <v-btn
+            text
+            :disabled="this.nicknameNotChanged()"
+            class="mb-2 text-subtitle-1 font-weight-bold amber--text text--accent-4"
+            @click.prevent="updateMember"
+          >
+            저장
+          </v-btn>
+        </div>
+      </div>
+    </transition>
+    <VSpacer />
+  </div>
+</template>
+
+<script>
+import api from "@/api/member";
+
+export default {
+  name: "PostCreateModal",
+  data() {
+    return {
+      rendered: false,
+      newNickname: "",
+    };
+  },
+  mounted() {
+    this.rendered = true;
+    this.newNickname = this.member.nickname;
+  },
+  computed: {
+    member() {
+      return this.$store.getters["member/getMember"];
+    },
+  },
+  methods: {
+    async updateMember() {
+      const updatedMember = await api.updateMember(this.newNickname);
+      this.$store.commit("member/SET_MEMBER", updatedMember);
+      this.$store.commit("snackbar/SHOW", "🥳 성공적으로 변경되었습니다.")
+      this.closeModal();
+    },
+    closeModal() {
+      this.$emit("close-modal");
+    },
```

Comment:
```suggestion
    close() {
      this.$emit("close");
    },
```
~Modal 내의 함수이니 그냥 close로 표현해도 괜찮을 것 같아요.